src/gui/guiFormSpecMenu.cpp

```diff
@@ -741,7 +741,8 @@ void GUIFormSpecMenu::parseImage(parserData* data, const std::string &element)
 		gui::IGUIImage *e = Environment->addImage(rect, this, spec.fid, 0, true);
 		e->setImage(texture);
 		e->setScaleImage(true);
-		e->setNotClipped(true);
+		auto style = getStyleForElement("image", spec.fname);
+		e->setNotClipped(style.getBool(StyleSpec::NOCLIP, false));
```

Comment:
Keeping backwards compatibility would be as easy as:
```suggestion
		e->setNotClipped(style.getBool(StyleSpec::NOCLIP, m_formspec_version < 3));
```

Reply:
Tested, works